### PR TITLE
gen-man-pages: command added to output man pages for acbuild

### PR DESCRIPTION
**Pull request by dgonyeo**
With this commit "acbuild gen-man-pages" will create the directory
./man/ and output a tree of man pages into this directory.

Fixes https://github.com/appc/acbuild/issues/205.

### CODE REVIEW

**Comment by dgonyeo** [comment]
Example output can be downloaded from here: https://csh.rit.edu/~dgonyeo/man-pages.tar.gz

**Comment by jonboulle** [comment]
Would be nice if this could be done without being baked into the binary - how about hiding it behind a tag and having a build-docs script that builds it with that tag?

**Comment by dgonyeo** [comment]
Just added the build tag `manpages` to `acbuild/man-page-generator.go`, and added the script `build-man-pages` that builds acbuild with that tag, calls `acbuild gen-man-pages`, and then deletes the compiled acbuild (so people don't accidentally start using the acbuild with man pages support).

File: build-man-pages [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# https://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+ACBUILD_BUILD_TAGS="-tags 'manpages'"
+
+source "${DIR}/build"
+
+${DIR}/bin/acbuild gen-man-pages
+
+# Don't want to leave acbuild with the gen-man-pages support laying around
```
Comment (jonboulle) [inline build-man-pages]: lying around

**Comment by jonboulle** [comment]
LGTM, thanks!

**Pull request merged**